src/components/virtualRepeat/virtual-repeater.js

```diff
@@ -243,7 +243,7 @@ VirtualRepeatContainerController.prototype.sizeScroller_ = function(size) {
 VirtualRepeatContainerController.prototype.autoShrink_ = function(size) {
   var shrinkSize = Math.max(size, this.autoShrinkMin * this.repeater.getItemSize());
   if (this.autoShrink && shrinkSize !== this.size) {
-    if (shrinkSize < (this.originalSize || this.size)) {
+    if (shrinkSize < (this.originalSize || this.size) || !(this.originalSize || this.size)) {
```

Comment:
That does have a different meaning, but the way that looks is still pretty awkward.